functions/Add-DbaAgReplica.ps1

```diff
@@ -133,6 +142,8 @@ function Add-DbaAgReplica {
         [DbaInstanceParameter[]]$SqlInstance,
         [PSCredential]$SqlCredential,
         [string]$Name,
+        [ValidateSet('Wsfc', 'External', 'None')]
+        [string]$ClusterType = (Get-DbatoolsConfigValue -FullName 'AvailabilityGroups.Default.ClusterType' -Fallback 'Wsfc'),
```

Comment:
I'm kind of thinking this should be `None` to avoid introducing breaking changes. WSFC has the highest amount of work frontloaded and I tend to create External or None.

Reply:
As we explicitly use the parameter when we call the command from New-DbaAvailabilityGroup, the default does not matter there.
But if someone later adds a replica (and uses a Windows Cluster), they would have to use this parameter to grant the correct rights. Having a default of Wsfc does the correct thing for most users. But grants more rights than needed for External an None. We should think about that.
I don't nee any breaking change here, can you specify? And I don't understand "WSFC has the highest amount of work frontloaded".

Reply:
ahhh, well im thinking about Linux clusters here, which does not support WSFC but does support `[NT AUTHORITY\SYSTEM]` to my knowledge -- it's called LocalSystem or something. I'll test to see.

> And I don't understand "WSFC has the highest amount of work frontloaded".

Basically, WSFC is the hardest to achieve, which makes me assume it won't be used as often as None. But that may just be an unfounded assumption.

Reply:
Oh, I think "in the real world" (and not our labs) every production Availability Group is based on a Windows Cluster. So I personally don't know any that is not based on a Windows Cluster. I think External and None are just implemented to get this working on Linux. But I may be completly wrong...